src/AspNetCoreModuleV2/CommonLib/FileOutputManager.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#pragma once
+
+
+class FileOutputManager : public IOutputManager
+{
+    #define FILE_FLUSH_TIMEOUT 3000
+public:
+    FileOutputManager(PCWSTR pwzStdOutLogFileName, PCWSTR pwzApplciationpath);
+    ~FileOutputManager();
+
+    virtual bool GetStdOutContent(STRU* struStdOutput) override;
+    virtual HRESULT Start() override;
+    virtual void NotifyStartupComplete() override;
+
+private:
+    HANDLE m_hLogFileHandle;
+    STTIMER m_Timer;
+    std::wstring m_wsStdOutLogFileName;
```

Comment:
i am not against std strings but those two can be replaced by STRU. This will keep the code consistent

Reply:
Can I make the whole file use wstrings instead?

Reply:
you need to do perf test to on STRU vs wstring if you want to totally replace one. the old claim is that STRU is better.